llvm/lib/Target/PowerPC/PPCFrameLowering.cpp

```diff
@@ -2607,6 +2658,11 @@ bool PPCFrameLowering::restoreCalleeSavedRegisters(
   unsigned CSIIndex = 0;
   BitVector Restored(TRI->getNumRegs());
 
+  Register MergeFrom = PPC::R31;
+  if (EnableLoadStoreMultiple && !Subtarget.isLittleEndian() &&
+      !Subtarget.isPPC64())
+    findContinuousLoadStore(CSI, MergeFrom);
```

Comment:
`findContinuousLoadStore` returns void, so `MergeFrom < PPC::R31` means the functions returns correctly, right? If so, we may need a comment here to explain why it's R31.